index.html

```diff
@@ -0,0 +1,257 @@
+                            </ul>
+                        </div>
+                        <div class="hover1">
+                            <ul>
+                                <li><a href="https://twitter.com/LewisHamilton/media">Media</a></li>
+                            </ul>
+                        </div>
+                        <div class="hover1">
+                            <ul>
+                                <li><a href="https://twitter.com/LewisHamilton/likes">Likes</a></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <div class="share">
+                        <div class="profileicon">
+                            <img src="https://pbs.twimg.com/profile_images/1387084860905476099/PIibmZi0_x96.jpg"
+                                alt="Lewis Hamilton's Profile Icon" width="48px" height="48px">
+                        </div>
+                        <div class="share-tweet">
+                            <div>
+                                <h2>What's happening</h2>
+                            </div>
+                            <div>Everyone can reply</div>
+                            <hr>
+                            <div>share3</div>
+                        </div>
+                    </div>
+                    <div class="feed post">
+                        <div class="feed-icon">
+                            <img src="https://pbs.twimg.com/profile_images/1387084860905476099/PIibmZi0_x96.jpg"
+                                alt="Lewis Hamilton's Profile Icon" width="48px" height="48px">
+                        </div>
+                        <div class="feed-tweet">
+                            <div class="tweeter">1</div>
+                            <div class="msg">2</div>
+                            <div class="media">3</div>
+                            <div class="tweet-stats">4</div>
+                        </div>
+                    </div>
+                    <div class="feed post">
+                        <div class="feed-icon">
+                            <img src="https://pbs.twimg.com/profile_images/1387084860905476099/PIibmZi0_x96.jpg"
+                                alt="Lewis Hamilton's Profile Icon" width="48px" height="48px">
+                        </div>
+                        <div class="feed-tweet">
+                            <div class="tweeter">1</div>
+                            <div class="msg">2</div>
+                            <div class="media">3</div>
+                            <div class="tweet-stats">4</div>
+                        </div>
+                    </div>
+                    <div class="feed post">
+                        <div class="feed-icon">
+                            <img src="https://pbs.twimg.com/profile_images/1387084860905476099/PIibmZi0_x96.jpg"
+                                alt="Lewis Hamilton's Profile Icon" width="48px" height="48px">
```

Comment:
setting height, width and other styles in the html is known as 'inline styling' it is best to avoid this where possible and use css to style things